src/test/java/com/epam/tc/hw2/ex1/HomePageTest.java

```diff
@@ -0,0 +1,75 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.WebDriverElement;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+@Test
+public class HomePageTest extends WebDriverElement {
+
+    @Test
+    public void testHomePage() {
+        SoftAssertions softAssertions = new SoftAssertions();
+        //1. open test site url
+        String url = "https://jdi-testing.github.io/jdi-light/index.html";
+        webDriver.navigate().to(url);
+        //2. assert browser title
+        softAssertions.assertThat(webDriver.getCurrentUrl()).isEqualTo(url);
+        softAssertions.assertThat(webDriver.getTitle()).isEqualTo("Home Page");
+        //3. perform login
+        webDriver.findElement(By.cssSelector("a[href='#']")).click();
+        webDriver.findElement(By.id("name")).sendKeys("Roman");
+        webDriver.findElement(By.id("password")).sendKeys("Jdi1234");
+        webDriver.findElement(By.id("login-button")).click();
+        //4. assert username is logged
+        WebElement webElement = webDriver.findElement(By.id("user-name"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("ROMAN IOVLEV");
+        //5. assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> elementsOnHeaderSection = webDriver.findElements(By.cssSelector(".m-l8 > li > a"));
+        softAssertions.assertThat(elementsOnHeaderSection).hasSize(4);
+        List<String> actualTextListElements = elementsOnHeaderSection.stream().map(WebElement::getText)
+            .collect(Collectors.toList());
+        List<String> expectedTextListElements = Arrays.asList("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS");
+        softAssertions.assertThat(actualTextListElements).isEqualTo(expectedTextListElements);
```

Comment:
Would be better to compare size with `expectedTextListElements.size()` instead of hardcoded `4`

\+ same for other checks of this type

Reply:
I corrected it